From ea45dd6f43ad1504ded7f2e2813997fa7d24fcc8 Mon Sep 17 00:00:00 2001
From: Samuel Dionne-Riel <samuel@dionne-riel.com>
Date: Fri, 10 Apr 2020 21:22:49 -0400
Subject: [PATCH] oneplus3: Configure LEDs using kernel triggers

 * The green LED is default-on
 * The red LED shows the device is being charged
---
 arch/arm64/boot/dts/15801_PVT/msm-pmi8994.dtsi | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/15801_PVT/msm-pmi8994.dtsi b/arch/arm64/boot/dts/15801_PVT/msm-pmi8994.dtsi
index b137c2bd7a5b7..df6174bdf35b0 100644
--- a/arch/arm64/boot/dts/15801_PVT/msm-pmi8994.dtsi
+++ b/arch/arm64/boot/dts/15801_PVT/msm-pmi8994.dtsi
@@ -649,8 +649,7 @@
 				qcom,pause-hi = <1000>;
 
 				linux,name = "red";
-				/*linux,default-trigger =
-					"battery-charging";*/
+				linux,default-trigger = "battery-charging";
 				qcom,use-blink;
 			};
 
@@ -661,7 +660,7 @@
 				pwms = <&pmi8994_pwm_2 0 0>;
 				qcom,pwm-us = <1000>;
 				qcom,max-current = <12>;
-				qcom,default-state = "off";
+				qcom,default-state = "on";
 				qcom,duty-pcts = [00 05 0a 0f 14 1d 28 32 3c 4b 64];
 				qcom,duty-ms = <20>;
 				qcom,start-idx = <13>;
@@ -673,7 +672,7 @@
 				qcom,pause-hi = <1000>;
 
 				linux,name = "green";
-				/*linux,default-trigger = "battery-full";*/
+				linux,default-trigger = "default-on";
 				qcom,use-blink;
 			};
 
-- 
2.23.1

